src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping/edit-keyboard-mapping.component.ts

```diff
@@ -65,6 +75,16 @@ export class EditKeyboardMappingComponent implements OnInit {
     }
   }
 
+  public buildKeyCombo($event: string[]): void {
```

Comment:
Methods shouldn't know anything about what is happening outside of its own scope. That means that method-, parameter- and variable-names shouldn't refer to things outside of the method.

You have named your parameter `$event`, but `buildKeyCombo` doesn't know that it is being used in a flow where there is an event. Also, there is nothing stopping me from writing:
```
public someMethodThatIsNotTriggeredByAnEvent(): void {
  const keys: string[] = ['KeyA']
  this.buildKeyCombo(keys)
}
```
You have named your parameter after _what you intend to parse to your method_ instead of _what is the parameter_. The parameter is an array consisting of keys, so it should simply be called `keys`. That it's currently being called by an event is not information that the method is privy to.

Sidenote: I would expoect that somethind calleed `buildX` returns something (just as I expect it for `createX`), but you return void.
Your method isn't really building anything. It's updating the "keys form" with new keys. So a possible name for it could be "updateKeys()" or something similar.

Reply:
Concerns addressed and committed to git (but not yet pushed)